runtime/common/src/precompile/incentives.rs

```diff
@@ -106,20 +106,11 @@ where
 				})
 			}
 			Action::GetDexRewardRate => {
-				let pool_currency_id = input.currency_id_at(1)?;
-				let pool_id = PoolId::Dex(pool_currency_id);
-
-				let value = <module_incentives::Pallet<Runtime> as IncentivesManager<
-					Runtime::AccountId,
-					Balance,
-					CurrencyId,
-					PoolId,
-				>>::get_dex_reward_rate(pool_id);
-
+				// NOTE: return default , or return PrecompileFailure?
```

Comment:
this is not enabled so should be safe to remove this
cc @zjb0807

Reply:
https://github.com/AcalaNetwork/Acala/blob/3f449446e4e2eea58582b958e8e1c967f301a55e/runtime/common/src/precompile/mod.rs#L136

`INCENTIVES` only enabled for mandala, it's safe

Reply:
let's remove this and the predeploy contract